src/lib/styles/styles.css

```diff
@@ -35,3 +35,18 @@
   --large-BorderRadius: 1rem;
   --large-focus-BorderRadius: 1.125rem;
 }
+
+.ui-regular-14,
+.ui-medium-14 {
+  font-family: 'IBM Plex Sans', sans-serif;
+  font-size: 0.875rem;
+  font-style: normal;
+  line-height: 1rem;
+  color: var(--global-primary-TextColor);
+}
+.ui-regular-14 {
+  font-weight: 400;
+}
+.ui-medium-14 {
+  font-weight: 500;
+}
```

Comment:
Where these classes come from? Where they are used? Also looks like they're same as default typography styles, so should not they inherit default theme typography values?

Reply:
I mean should not we move typography stuff to `styles.css` in a way like below?

```css
 /** Typography **/

* {
    font-family: 'IBM Plex Sans', sans-serif;
    font-size: 0.875rem;
    font-style: normal;
    font-weight: 400;
    line-height: 1rem;
}
```

or `font: inherit;` in the button